x/qgb/README.md

```diff
@@ -110,56 +380,32 @@ if err != nil {
+### Hooks initialization panic
 
-```shell
-$ celestia-appd verify --help          
-                              
-Verifies that a transaction hash, a range of shares, or a blob referenced by its transaction hash were committed to by the QGB contract
-
-Usage:
-  celestia-appd verify [command]
-
-Available Commands:
-  blob        Verifies that a blob, referenced by its transaction hash, in hex format, has been committed to by the QGB contract.
-  shares      Verifies that a range of shares has been committed to by the QGB contract
-  tx          Verifies that a transaction hash, in hex format, has been committed to by the QGB contract
-
-Flags:
-  -h, --help   help for verify
-
-Global Flags:
-      --home string         directory for config and data (default "/home/midnight/.celestia-app")
-      --log_format string   The logging format (json|plain) (default "plain")
-      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
-      --trace               print out full stack trace on errors
-
-Use "celestia-appd verify [command] --help" for more information about a command.
+When initializing the QGB hooks, if the QGB store key is not setup correctly, the state machine will [panic](https://github.com/celestiaorg/celestia-app/blob/0629c757ef35a24187a8d7a4c706c7cdc894c8b6/x/qgb/keeper/hooks.go#L14-L19):
 
+```golang
+// if startup is mis-ordered in app.go this hook will halt the chain when
+// called. Keep this check to make such a mistake obvious
+if k.storeKey == nil {
+panic("hooks initialized before GravityKeeper")
+}
 ```
 
-It currently supports three sub-commands:
-
-- `blob`: Takes a transaction hash, in hex format, and verifies that the blob paid for by the transaction has been committed to by the QGB contract. It only supports one blob for now.
-- `shares`: Takes a range of shares and a height, and verifies that these shares have been committed to by the QGB contract.
-- `tx`: Takes a transaction hash, in hex format, and verifies that it has been committed to by the QGB contract.
-
-## Useful links
+## Appendix
 
 The smart contract implementation is in [quantum-gravity-bridge](https://github.com/celestiaorg/quantum-gravity-bridge/).
 
-For QGB v2, the orchestrator and relayer implementations will be in a separate repository. Links will be added subsequently.
+For QGB v2, the orchestrator and relayer implementations are in the [orchestratory-relayer](https://github.com/celestiaorg/orchestrator-relayer) repo.
 
-QGB v1 implementation, including the orchestrator and relayer, is in the [qgb-integration](https://github.com/celestiaorg/celestia-app/tree/qgb-integration) branch.
+QGB v1 implementation, including the orchestrator and relayer, is in the [qgb-integration](https://github.com/celestiaorg/celestia-app/tree/qgb-integration) branch (deprecated).
```

Comment:
do we provide other information on versions elsewhere? afaik, I think we're the only ones who know what v1 or v2 are :sweat_smile: . If we don't distinguish between v1 and v2 in the code, we might want to only reference off hand here.

if we want to document the iterations we've done, then we might want to just do that in a short paragraph here, or in a separate document with more details. or maybe reference old ADRS or something not sure

Reply:
sounds good

Reply:
[cca1dab](https://github.com/celestiaorg/celestia-app/pull/1911/commits/cca1dab597b78af7c4e6d38a624e906301e7a141)